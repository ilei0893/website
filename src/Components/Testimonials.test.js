import React from "react";
import Testimonials from "./Testimonials.js";
import renderer from "react-test-renderer";

describe("Tests the testimonials section", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<Testimonials />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
