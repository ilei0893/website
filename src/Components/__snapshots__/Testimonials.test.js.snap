// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests the testimonials section matches the snapshot 1`] = `
<section
  id="testimonials"
>
  <div
    className="text-container"
  >
    <div
      className="row"
    >
      <div
        className="two columns header-col"
      >
        <h1>
          <span>
            Client Testimonials
          </span>
        </h1>
      </div>
      <div
        className="ten columns flex-container"
      >
        <ul
          className="slides"
        />
      </div>
    </div>
  </div>
</section>
`;
